src/AgApiClient.cpp

```diff
@@ -130,7 +130,7 @@ bool AgApiClient::postToServer(String data) {
   client.end();
 
   logInfo(String("POST: ") + uri);
-  logInfo(String("DATA: ") + data);
+  // logInfo(String("DATA: ") + data);
```

Comment:
propose to continue to log the data

Reply:
We already log `AgValue::toString()` return value [here](https://github.com/airgradienthq/arduino/blob/feat/particle-count/src/AgValue.cpp#L508). So it will not printout when HTTP POST occur, but also for MQTT and local server call.
What do you think?